jobs/collect_logs.sh

```diff
@@ -3,10 +3,10 @@ source /usr/local/src/cinder-ci/jobs/utils.sh
 
 echo "Collecting logs"
 
-if [ $DEBUG_JOB = "yes" ]; then
-    LOGSDEST="/srv/logs/cinder/debug/$ZUUL_CHANGE/$ZUUL_PATCHSET/$JOB_TYPE"
-else
+if [ $DEBUG_JOB != "yes" ] || [ -z "$DEBUG_JOB" ]; then
```

Comment:
did you actually test this: [ $DEBUG_JOB != "yes" ] ?
At the same time, once you test that it's different from "yes" why do you even bother to test the "-z" part?